File: app/src/main/java/moe/tlaster/openween/core/model/comment/CommentListModel.java
package moe.tlaster.openween.core.model.comment;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import moe.tlaster.openween.core.model.BaseListModel;

/**
 * Created by Tlaster on 2016/9/2.
 */
public class CommentListModel extends BaseListModel {
    @SerializedName("comments")
    private List<CommentModel> mComments;

    public List<CommentModel> getComments() {
        return mComments;
    }

    public void setComments(List<CommentModel> comments) {
        mComments = comments;
    }
}

File: app/src/main/kotlin/moe/tlaster/openween/core/model/comment/CommentListModel.kt
package moe.tlaster.openween.core.model.comment

import com.google.gson.annotations.SerializedName

import moe.tlaster.openween.core.model.BaseListModel

/**
 * Created by Tlaster on 2016/9/2.
 */
class CommentListModel : BaseListModel() {
    @SerializedName("comments")
    var comments: List<CommentModel>? = null
}
